src/preact/context.js

```diff
@@ -90,19 +90,23 @@ export function useAmpContext() {
  * Whether the calling component should currently be in the loaded state.
  *
  * @param {!Loading|string} loadingProp
+ * @param {boolean} unloadOnPause
  * @return {boolean}
  */
-export function useLoad(loadingProp) {
+export function useLoad(loadingProp, unloadOnPause = false) {
   const loadingRef = useRef(false);
 
-  const {loading: loadingContext, renderable} = useAmpContext();
+  const {loading: loadingContext, renderable, playable} = useAmpContext();
 
   const loading = loadingReducer(loadingProp, loadingContext);
 
   // Compute whether the element should be loading at this time.
   const load =
     // Explicit instruction to unload.
     loading == Loading.UNLOAD
+      ? false
```

Comment:
Can this be simplified to `loading == Loading.EAGER && (!unloadOnPause || playable)`?

Reply:
Can simply, but I'd rather let compiler do that because I think these are importantly two different use cases. But compiler will most likely combine them as it pleases.

Reply:
sounds good